settingService/blockUser.go

```diff
@@ -78,20 +77,15 @@ type ContactResult struct {
 }
 
 // IsUserBlocked checks if user is blocked for chatting in contacts table
-// Function returns: if succeed - is_blocked value(0 or 1) from contacts table and nil, if failed - nil and error
-func IsUserBlocked(byteRequest [] byte) (isBlocked int, err error) {
-	request, err := messageService.UnmarshalMessage(byteRequest)
-	if err != nil {
-		loger.Log.Errorf("Error has occurred: ", err)
-		return nil, err
-	}
+// Function returns: if succeed - true or false from contacts table(depend is contact blocked or not) and nil, if failed - nil and error
+func IsUserBlocked(request *messageService.Message) (isBlocked bool, err error) {
+	isBlocked = true
 	contactUser := request.Body.ReceiverName
```

Comment:
Unresolved reference 'ReceiverName'
Body has type RawMessage. So you cannot get ReceiverName in this way